docs/reference/workflows/monobehaviour/creating-workers.md

```diff
@@ -17,31 +22,29 @@ When you have multiple worker instances represented as GameObjects in a Scene, y
 
 #### Scripts
 
 The scripts are stored inside `workers/unity/Assets/Scripts/Workers`:
 
 * `UnityClientConnector`: This is a sample implementation to connect as a client-worker on Windows or MacOS. (Note that the client-worker is sometimes called `UnityClient`.)
 * `UnityGameLogicConnector`: This is a sample implementation to connect as a server-worker.
-* `AndroidClientWorkerConnector`: This is a sample implementation to connect as a client-worker on an Android device.
-* `iOSClientWorkerConnector`: This is a sample implementation to connect as a client-worker on an iOS device.
+* `MobileClientWorkerConnector`: This is a sample implementation to connect as a client-worker on an Android or iOS device.
 
 #### Scenes
 
 * `ClientScene`: This Scene contains only the `ClientWorker` GameObject which you can use to build a client-worker instance for cloud deployments (for a game client on Windows or MacOS)
-* `AndroidClientScene`: This Scene contains only the `AndroidClient` GameObject which you can use to build your client-worker for local and cloud deployments (for a game client on Android).
-* `iOSClientScene`: This Scene contains only the `iOSClient` GameObject which you can use to build your client-worker for local and cloud deployments (for a game client on iOS).
+* `MobileClientScene`: This Scene contains only the `ClientWorker` GameObject which you can use to build your client-worker for local and cloud deployments (for a game client on Android or iOS).
 * `GameLogicScene`: This Scene contains only the `GameLogicWorker` GameObject which you can use to build a server-worker instance for cloud deployments (as it’s a server-worker instance, the game client platform you are using is irrelevant).
-* `DevelopmentScene`: This Scene contains both the `ClientWorker` GameObject representing the client-worker instance and the `GameLogicWorker` GameObject representing the server-worker instance and starts both worker instances as soon as you load the Scene.
+* `DevelopmentScene`: This Scene contains both the `ClientWorker` GameObject representing the client-worker instance (on Windows or MacOS) and the `GameLogicWorker` GameObject representing the server-worker instance and starts both worker instances as soon as you load the Scene.
 
 ## How to create your own WorkerConnector
 
 You can inherit from the [`WorkerConnector`]({{urlRoot}}/api/core/worker-connector) class to create your own connection logic, dependent on the [type of the worker]({{urlRoot}}/reference/glossary#worker-types) that you want to create.
 
 **Example**</br>
-Showing what your implementation, inheriting from [`DefaultWorkerConnector`]({{urlRoot}}/api/core/default-worker-connector), could look like.
+Showing what your implementation, inheriting from the [`DefaultWorkerConnector`]({{urlRoot}}/api/core/default-worker-connector), could look like.
```

Comment:
this text + below code could be an expandable

text could also be trimmed to something like "Example worker connector implementation, inheriting from the `DefaultWorkerConnector`."